plugins/modules/nxos_vrf_global.py

```diff
@@ -0,0 +1,419 @@
+                    type: dict
+                    suboptions:
+                      resilient:
+                        description: Configure resilient RPF interface.
+                        type: bool
+                      splitting_type:
+                        description: Configure multicast load splitting type.
+                        type: dict
+                        mutually_exclusive:
+                          [["none", "sg_hash", "sg_hash_next_hop"]]
+                        suboptions:
+                          none:
+                            description: Disable multicast load splitting.
+                            type: bool
+                          sg_hash:
+                            description: Configure hash based on source and group address.
+                            type: bool
+                          sg_hash_next_hop:
+                            description: Configure hash based on source and group address and next-hop.
+                            type: bool
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the NX-OS device
+        by executing the command B(show running-config | section ^vrf).
+      - The state I(parsed) reads the configuration from C(running_config) option and
+        transforms it into Ansible structured data as per the resource module's argspec
+        and the value is then returned in the I(parsed) key within the result.
+    type: str
+  state:
+    choices:
+      [
+        parsed,
+        gathered,
+        deleted,
+        merged,
+        replaced,
+        rendered,
+        overridden,
+        purged,
+      ]
+    default: merged
+    description:
+      - The state the configuration should be left in
+      - The states I(rendered), I(gathered) and I(parsed) does not perform any change
+        on the device.
+      - The state I(rendered) will transform the configuration in C(config) option to
+        platform specific CLI commands which will be returned in the I(rendered) key
+        within the result. For state I(rendered) active connection to remote host is
+        not required.
+      - The state I(gathered) will fetch the running configuration from device and transform
+        it into structured data in the format as per the resource module argspec and
+        the value is returned in the I(gathered) key within the result.
+      - The state I(parsed) reads the configuration from C(running_config) option and
+        transforms it into JSON format as per the resource module parameters and the
+        value is returned in the I(parsed) key within the result. The value of C(running_config)
+        option should be the same format as the output of command I(show running-config | section ^vrf).
+        connection to remote host is not required.
+    type: str
+"""
+
+EXAMPLES = """
```

Comment:
Can we also populate the examples

Reply:
Sure, i have to start working on config side code this sprint, will update them